packages/ui/src/components/Menu/Menu.stories.tsx

```diff
@@ -7,38 +7,32 @@ export default {
 } as ComponentMeta<typeof Menu>;
 
 export const Default: ComponentStory<typeof Menu> = () => (
-  <div className="w-64 h-32 flex flex-row-reverse">
```

Comment:
Removed these `div`s because it was messing with the styles. 

This is probably indicative of another problem - the Menu styling isn't self-contained.

Reply:
This is probably the reason the menus had `relative` and `absolute`. 
I'll see if I can fix this up. If you want, I can reverse this change.